starboard/tools/testing/test_runner.py

```diff
@@ -265,7 +265,7 @@ def __init__(self,
     _VerifyConfig(self._platform_config,
                   self._platform_test_filters.GetTestFilters())
 
-    if self.loader_platform:
+    if self.loader_platform and self.loader_platform.find("rdk") == -1:
```

Comment:
We don't want platform specific code like this in the test runner. I think the best way to handle this is probably to add some logic to the build config (`gyp_configuration.py`). It's fetched about on line 250.

Reply:
Removed the platform specific check from test_runner.